pages/about/groups/ariawg.md

```diff
@@ -33,66 +31,114 @@ Some information on this page is also shown on, and may be more current in, the
 {% include_cached toc.html type="end" %}
 {:/}
 
-## Announcements
+## Charter
+
+The ARIA Working Group is chartered until January 2027.
+
+To learn about the group’s focus, scope and deliverables, see the [ARIA Working Group Charter](https://www.w3.org/2025/01/aria-charter).
+
+## Current work 
+
+For details of the current work, see the [ARIA Working Group README on GitHub](https://www.github.com/w3c/aria/).
+
+The ARIA Working Group follows a set process for developing ARIA guidelines and resolving ARIA issues &emdash; for details, see [ARIA Working Group Process Document](https://github.com/w3c/aria/blob/main/documentation/process.md).
+
+### WAI-ARIA
+
+The [WAI-ARIA specification](https://www.w3.org/TR/wai-aria/) defines roles, states, and properties that can be used in a variety of host languages to enhance the accessibility of user interfaces.
+
+### WAI-ARIA modules
+
+The ARIA modules extend the roles, states, and properties for specific host languages, such as:
+
+* [Digital Publishing](https://www.w3.org/TR/dpub-aria-1.1/)
+* [Graphics](https://www.w3.org/TR/graphics-aria-1.0/)
+* [HTML](https://www.w3.org/TR/html-aria/)
+
+### ARIA Core Accessibility API Mappings
+
+The ARIA [Core Accessibility API Mappings](https://www.w3.org/TR/core-aam-1.2/) describe how user agents should expose semantics of web content languages to accessibility APIs. The mappings apply broadly to multiple technologies. In addition, the ARIA Working Group develops specific mappings for:
+
+* CSS
+* [Digital Publishing](https://www.w3.org/TR/dpub-aam-1.1/)
+* [Graphics](https://www.w3.org/TR/graphics-aam/)
+* [HTML](https://www.w3.org/TR/html-aam-1.0/)
+* MathML
+* PDF
+* [SVG](https://www.w3.org/TR/svg-aam-1.0/)
```

Comment:
@tamsinewing555Note that I am not adding links to any of these two because they are still Working Drafts